.github/workflows/release_docker_gateway.yaml

```
@@ -0,0 +1,44 @@
+permissions:
+    contents: write
+    pull-requests: write
+
+jobs:
+    release-image:
+        name: Release Gateway Docker Image
+        runs-on: ubuntu-latest
+        steps:
+            - name: Extract version from tag
+              id: extract_version
+              run: echo "::set-output name=version::${GITHUB_REF_NAME#infisical-gateway/}"
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)